### DISCUSSION TITLE
LORAWAN Compatibility with nrf52832 and sx1262

### DISCUSSION THREAD

**Discussion by Ahsan-Averos** [zephyrproject-rtos/zephyr#35829] [General]
Dear All.
I am trying to integrate Lorawan on **"NRF52Dk"** with **"Sx1262"**(E22900M22S) module.
Can someone please helping me out, how can I use Zephyr Lorawan with my project. (if it is possible with Zephyr).
I saw number of project done with SX1276 with NRF52 in Zephyr but this project is not in detail.
Can someone can guide me step by step. I will be very highly thank full for your support!
Thanks in advance

**Comment by carlescufi** [comment]
@Ahsan-Averos I have converted this to a discussion since it is more of a request for help than an actual bug or enhancement report.
